integration_tests/gdk/components/test_integ_BuildCommand.py

```diff
@@ -1,7 +1,7 @@
 import json
 from pathlib import Path
 from shutil import Error
-from unittest.mock import mock_open, patch
+from gdk.commands.component.transformer.BuildRecipeTransformer import BuildRecipeTransformer
```

Comment:
If you re export on the __init__.py you can avoid retyping the names

```python
#__init__.py
from .BuildRecipeTransformer import BuildRecipeTransformer
```

Then you imports here could be `from gdk.commands.component.transformer import BuildRecipeTransformer`